### ISSUE TITLE
New styling TUe house style powerpoint

### ISSUE THREAD

**Issue by JoostvanPinxten** [Caster/Beamer-TUe#14]
There is a new house style out, as of [July 16th](https://studiegids.tue.nl/nieuws-agenda/nieuws/16-07-2015-nieuwe-styling-tue-powerpoint-presentaties/). It would be nice if you could translate these into their Latex equivalent too!

The slides in the new housestyle can be found [here](https://intranet.tue.nl/universiteit/diensten/communicatie-expertise-centrum/huisstijl-middelen/huisstijlmiddelen/powerpoint-presentaties/).

**Comment by Caster** [comment]
Yes, that is definitely the plan. I plan on working on this in the coming weeks.

**Comment by Willem3141** [comment]
We are working on it; see #15 :smile:
